templates/extensions/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+CHANGELOG.md is how you tell users what has changed in each version of your extension.
+When you release a new version, add a new header and release notes for that version.
+When users update their instance, they will see the release notes for 
```

Comment:
incomplete sentence?

Reply:
Sorry, forgot to hit ctrl-s one last time before committing 😬